src/isPlainObject.ts

```diff
@@ -2,7 +2,7 @@
  * @param {any} obj The object to inspect.
  * @returns {boolean} True if the argument appears to be a plain object.
  */
-export default function isPlainObject(obj) {
+export default function isPlainObject(obj: any): obj is object {
```

Comment:
`obj: unknown`

Reply:
Thanks, I'm not used to the new `unknown` type yet.